src/app/components/ThemeProvider/index.tsx

```diff
@@ -19,7 +19,10 @@ export const ThemeProvider = ({
 
   const serviceVariant: Variants = variant || defaultServiceVariants[service];
 
-  if (serviceVariant === 'default' || !serviceVariant) {
+  if (
+    (serviceVariant === 'default' && service !== 'uzbek') ||
+    (!serviceVariant && service !== 'uzbek')
+  ) {
```

Comment:
What happens if we keep the original check? I'd imagine it _may_ load in both themes for Uzbek, which will probably have knock-on effects if the font family changes for `lat`.